collections/_blogs/2020-08-04-how-i-schooled-my-seniors.md

```diff
@@ -0,0 +1,26 @@
+
+I pitched my idea to [Lee Calcote](https://slack-redir.net/link?url=https%3A%2F%2Ftwitter.com%2Flcalcote&v=3) and in his usual supportive form, he was as excited as I was about the session. A brainstorm session led to creating an entire presentation with suggestions from Lee. The Layer5 community offers a number of resources to contributors, including these two decks ([Meshery - The Service Mesh Management Plane](https://slack-redir.net/link?url=https%3A%2F%2Fdocs.google.com%2Fpresentation%2Fd%2F14kxjwYSJ_FyE3K_6CDEd6oq2kqwn0OSE8RDJ4H-KlKU%2Fedit%3Fusp%3Dsharing&v=3) and [Layer5 - The Service Mesh Community](https://slack-redir.net/link?url=https%3A%2F%2Fdocs.google.com%2Fpresentation%2Fd%2F1tLcMcyQ-Z6YYISYyR0Eyb30dojQiKHIjs7Rt5m8kcag%2Fedit%3Fusp%3Dsharing&v=3)) that I drew from as a basis for a personalized presentation and demonstration to my new team.
+
+<center>
+<iframe src="https://docs.google.com/presentation/d/e/2PACX-1vQNpnp_SIkYNMe1rx-Cw_armxHRmQvkcp8d3BVML6XEhsSgy7k0w9xDKrGGs4EPew/embed?start=true&loop=true&delayms=3000" frameborder="0" width="100%" height="300" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe></center>
+
+When the day arrived to present the concepts of service meshes, Meshery, and other cloud native initiatives to my team, I held a Google Meet with a full attendance. I started the session with a reader’s digest version of microservices and service meshes to set the stage for any newcomers to the world of cloud native, I went on to elaborate on Layer5 ‘s contributions to the CNCF landscape and the SMI project. This was followed by an in-depth discussion on data planes, control planes, and management planes. I spoke at-length about Meshery’s functionality including, its features for facilitating the benchmarking, interoperability and standardization of service meshes. I wrapped up the session with some insider tips that I have learned in my months as an open source contributor in the Layer5 community. In the stream of feedback received from my colleagues, the session seems to have been a success.
+
+This whole experience has been an amazing journey for me. From being a complete noob to utilizing my expertise in the service mesh domain to landing an actual job, the experience has been a maze of ups and downs. But, I feel like I have finally arrived and arrived with a pot of gold. I’m eternally grateful to Lee Calcote for motivating me and providing me with the appropriate resources. Entering and exploring an open source community can be a daunting task and the [resources provided by Layer5](https://slack-redir.net/link?url=https%3A%2F%2Fdrive.google.com%2Fdrive%2Ffolders%2F1Wg_LPY3ImAexejNFpHWMCBr26UiMUK68%3Fusp%3Dsharing&v=3) helped them in overcoming that challenge and enabled them to learn more about the widespread work of [Meshery](https://slack-redir.net/link?url=https%3A%2F%2Fmeshery.io%2F&v=3).
```

Comment:
Link should be directly to Google Drive not via Slack
@chandrashritii Am I right?

Reply:
Right, no, it shouldn't.

Reply:
Thanks for pointing it out @NupurThakur27. I'll correct it